### PR TITLE
Model Variable Naming Sprint Batch 2

### PR DESCRIPTION
**Pull request by mikecooke77**
The following changes have been made:

`specific_humidity -> water_vapor_mixing_ratio_wrt_moist_air`
 OR
`specific_humidity -> water_vapor_mixing_ratio_wrt_moist_air_and_condensed_water`

`specific_humidity_at_two_meters_above_surface -> water_vapor_mixing_ratio_wrt_moist_air_at_2m`
OR
`specific_humidity_at_two_meters_above_surface -> water_vapor_mixing_ratio_wrt_moist_air_and_condensed_water_at_2m`

I have made the choices I think are correct based on UKMO using "and_condensed_water" and other partners not using this.

### CODE REVIEW

**Review by matthewrmshin — APPROVED** [review]
Tested in my environment. Works as expected.

File: update_part2.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
+'''
```
Comment (ctgh) [inline update_part2.py:1]: Did you mean to upload this? No problem if so.

**Review by ctgh — APPROVED** [review]
Looks good!

**Comment by svahl991** [comment]
@mikecooke77 When/if you have time, can you resolve the merge conflicts here and re-trigger the UFO tests?

**Review by danholdaway — APPROVED** [review]

**Pull request merged**